oper8/deploy_manager/openshift_deploy_manager.py

```diff
@@ -743,20 +743,33 @@ def _apply_resource(self, resource_definition: dict) -> dict:
                 field_manager="oper8",
             ).to_dict()
         except ConflictError:
-            log.debug(
-                "Overriding field manager conflict for [%s/%s/%s] in %s ",
-                api_version,
-                kind,
-                name,
-                namespace,
-            )
-            return resource_handle.server_side_apply(
-                resource_definition,
-                name=name,
-                namespace=namespace,
-                field_manager="oper8",
-                force_conflicts=True,
-            ).to_dict()
+            try:
+                log.debug(
+                    "Overriding field manager conflict for [%s/%s/%s] in %s ",
+                    api_version,
+                    kind,
+                    name,
+                    namespace,
+                )
+                return resource_handle.server_side_apply(
+                    resource_definition,
+                    name=name,
+                    namespace=namespace,
+                    field_manager="oper8",
+                    force_conflicts=True,
+                ).to_dict()
+            except UnprocessibleEntityError as err:
```

Comment:
I probably missed some of the discussion on this, but why does this only happen _after_ the initial `ConflictError`?

Reply:
So the initial `ConflictError` is raised because SSA detects that another controller has taken ownership of a field (e.g. ManagedFields has a non oper8 owner). So we then reapply with `force_conflicts=True` but that runs into the 422 due to env vars